gather_cnv_v2v

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+
+for ns in $(oc get namespace $ns --no-headers -o=custom-columns=NAME:.metadata.name 2> /dev/null)
+do
+    echo "NS: $ns"
+    ns_collection_path=$BASE_COLLECTION_PATH/namespaces/$ns/
+    mkdir -p $ns_collection_path
+
+    for pod in $(oc get pods -n $ns --no-headers -o=custom-columns=NAME:.metadata.name 2>/dev/null)
+    do
+        if ! expr $pod : 'kubevirt-v2v-conversion-*' >/dev/null
```

Comment:
Suggestion: I would label all my v2v pods with something, and then use -l in the for loop to pre-filter here. CDI uses cdi.kubevirt.io to label all our pods.